app/konnect/runtime-manager/index.md

```diff
@@ -45,4 +45,12 @@ choose to manually configure runtime instances on the following:
 * AWS
 * Azure
 
-See the [runtime instance installation options](/konnect/runtime-manager/runtime-instances/) for more detail.
\ No newline at end of file
+See the [runtime instance installation options](/konnect/runtime-manager/runtime-instances/) for more detail.
+
+## Plugins
+
+You can extend {{site.konnect_short_name}} by using plugins. Kong provides a set of standard Lua plugins that get bundled with {{site.konnect_short_name}}. The set of plugins you have access to depends on your installation.
+
+Custom plugins can also be developed by the Kong Community and are supported and maintained by the plugin creators. If they are published on the Kong Plugin Hub, they are called Community or Third-Party plugins.
```

Comment:
This needs a point specifically about Konnect plugins. And how Konnect supports a series of plugins that were supported in the latest release.

Reply:
Maybe direct link the the list in your `/plugins/` doc.